userguide/install.rst

```diff
@@ -298,9 +298,14 @@ system.
 
 Use the arrow keys to highlight the destination SSD, hard drive,
 |usb-stick|, or virtual disk. Press the :kbd:`spacebar` to select
-it. To mirror the |os-device|, move to the second device and press
-:kbd:`spacebar` to select it also. After making these selections,
-press :kbd:`Enter`. The warning shown in
+it.
+
+To mirror the |os-device|, move to the second device and press
+:kbd:`spacebar` to select it also. 16 GiB swap partitions are created
```

Comment:
It's not clear what the second sentence is saying.  I think it means:
"If the device is not a USB device, and larger than 64 GiB, a 16 GiB swap partition is also created on it."